Agreements/Internal/Solent_TCSW.txt

```diff
@@ -1,9 +0,0 @@
-COPX:EGHI:*:*:NORRY:*:SOLENT:London TC SW:10000:*:^NORRY
-
-COPX:EGHI:*:TABEN:*:*:SOLENT:London TC SW:10000:*:^TABEN
-COPX:EGHH:*:TABEN:*:*:SOLENT:London TC SW:10000:*:^TABEN
-
-COPX:EGHI:*:PEPIS:CPT:*:SOLENT:London TC SW:10000:*:^PEPIS
-
-COPX:EGHI:*:GWC:*:*:SOLENT:London TC SW:10000:*:^GWC
-COPX:EGHH:*:GWC:*:*:SOLENT:London TC SW:10000:*:^GWC
```

Comment:
Don't need this

Reply:
I don't know why that is there, but I cannot find a way to remove it. I've tried roll back etc. with no luck.

Reply:
@luketp try creating the file again with the same contents and push to the branch you made the changes on?